pkg/ccn-lite/Makefile.dep

```diff
@@ -4,3 +4,4 @@ USEMODULE += evtimer
 USEMODULE += random
 USEMODULE += timex
 USEMODULE += posix_headers
+USEMODULE += xtimer
```

Comment:
Can you rebase? This is no longer needed since it was migrated to ztimer.

Reply:
https://github.com/cn-uofbasel/ccn-lite/blob/da0d9de8d82349dff845acc62d37242dd09b3d3d/src/ccnl-riot/src/ccn-lite-riot.c

Reply:
Nope, b77b21ba137df8cf5ea656996e6fdf07411f44ff

Reply:
it was true when i wrote the comment 14:07 is earlier that 14:10

Reply:
Yep sorry, I though it was already merge, but no, it was just green and ready to go